RabbitMQ.pm

```diff
@@ -330,25 +345,32 @@ C<$props> is the hash sent by publish()  respecting the following keys:
        timestamp => $integer,
      }
 
-If you provide a timeout, then the C<recv()> method returns C<undef> if the
+C<$timeout> is a positive integer, specifying the number of milliseconds to
+wait for a message. If you do not provide a timeout (or set it to 0), then
+this call will block until it receives a message. If you set it to -1 it will
+return immediately (waiting 0 ms).
+
+If you provide a timeout, then the C<recv> method returns C<undef> if the
 timeout expires before a message is received from the server.
 
 =head2 cancel($channel, $consumer_tag)
 
+This method returns true or false indicating whether we got the expected
+"cancel-ok" response from the server.
```

Comment:
Sure! The `cancel` call actually pulls a channel out of `consume` mode. More docs are available here: https://www.rabbitmq.com/consumer-cancel.html